Cython/Build/pep517_backend/_backend.py

```diff
@@ -0,0 +1,142 @@
+# Note - setuptools imports are all lazy so that we can specify it as a dependency
+from contextlib import contextmanager as _contextmanager
+from functools import lru_cache
```

Comment:
This name should also be private so that re-export in the init wouldn't pick it up by default.